### PR TITLE
Master adding emojipicker to editor shsa

### PR DESCRIPTION
**Pull request by shsa-odoo**
Purpose:

This commit adds Emoji_picker functionality to the web_editor,
allowing users to conveniently add emojis to their text and
improving the overall user experience.

Specifications:

Now we can insert emoji in web_editor using /emoji.

Task id: 3253958

### CODE REVIEW

**Review by alexkuhn — COMMENTED** [review]
LGTM

File: addons/web/static/src/core/emoji_picker/emoji_picker.js [hunk 1]
```diff
@@ -149,7 +149,7 @@ export class EmojiPicker extends Component {
                 this.emojis.map((emoji) => [emoji.codepoints, emoji])
             );
             this.state.categoryId = this.categories[0]?.sortId;
-            this.recent = JSON.parse(browser.localStorage.getItem("mail.emoji.frequent") || "{}");
+            this.recent = JSON.parse(browser.localStorage.getItem("web.emoji.frequent") || "{}");
```
[the remark this answers is not in the document]
Comment (shsa-odoo) [reply addons/web/static/src/core/emoji_picker/emoji_picker.js:152]: Thankyou @alexkuhn :clinking_glasses:

**Comment by SimonGenin** [comment]
@robodoo r+

**Comment by SimonGenin** [comment]
@robodoo rebase-ff

**Pull request closed without merging**